marketplace/sensitive-language/src/no-issues.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import React from 'react';
+
+import {Icon, Typography, Subheading, Paragraph} from '@contentful/forma-36-react-components';
+
+export function NoIssues() {
+  return (
+    <Typography>
+      <Subheading style={{display: 'flex', alignItems: 'center'}}>
+        <Icon icon="CheckCircle" color="positive" extraClassNames="f36-margin-right--xs" />
```

Comment:
I see too many to ways to apply styles here: `Typography` component, inline styles and forma utility classes. Would be better just to use CSS in this case.

Reply:
@m10l Since you wrote this part, WDYT?